scripts/generate-config-data.js

```diff
@@ -117,7 +127,7 @@ function makeCliConfigTree (parsedCliOptions) {
 function getStructFields (name, source) {
   const subsource = source.match(new RegExp(`^struct ${name} {[^]+?}`, 'm'))[0];
 
-  const fields = subsource.split('\n').slice(1, -1);
+  const fields = subsource.split('\n').slice(1, -1).filter(s => s.trim() && !s.trim().startsWith('#') && !s.trim().startsWith('_legacy'));
```

Comment:
The first `trim()` looks superfluous. How about: `s => s=s.trim() && !s.startsWith(…) && !s.startsWith(…)`?

Reply:
needs to match for example

```
	#[serde(rename="public_node")]
```

so we first need to trim() to remove the indentation and then check the starting chracters